tests/test_biolink_model/test_biolink_model.py

```diff
@@ -19,6 +19,8 @@
 
 BIOLINK_NS = Namespace("https://w3id.org/biolink/vocab/")
 
+pytestmark = pytest.mark.biolink
+
 
 @pytest.mark.slow
```

Comment:
Why not use the annotation directly ? 😇 

```suggestion


@pytest.mark.biolink
@pytest.mark.slow
```